stf_functional_tests.yml

```diff
@@ -52,6 +52,21 @@
       import_role:
         name: client_side_tests
         tasks_from: test_e2e
+      
+    - name: "Test Alerts"
+      import_role:
+        name: test_alerts
+        tasks_from: first_test
+    
+    - name: "Test metrics retention"
+      import_role:
+        name: test_metrics_retention
+        tasks_from: metrics_retention_test
+
+    - name: "Test snmpTraps"
+      import_role:
+        name: test_snmpTraps
+        tasks_from: snmpTraps_test
```

Comment:
ditto here.
There's one task in the role at the moment, you can rename ``roles/tasks/snmpTraps_test.yml`` to ``roles/tasks/main.yml`` and remove the ``tasks_from`` line